src/graph/test/GoTest.cpp

```diff
@@ -917,6 +917,25 @@ TEST_F(GoTest, ReverselyOneStep) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    {
+        cpp2::ExecutionResponse resp;
+        auto query = "GO FROM hash('Tim Duncan') OVER * REVERSELY "
+                     "YIELD like._src";
```

Comment:
Would you please add an UT with multi-edges?  I am not quite sure if this test case could cover it.